tests/unittests/dataset_tests.py

```
@@ -1148,6 +1148,89 @@ def map_fcn(sample):
 
         self.assertDictEqual(dict(counter), {"BAR": 50})
 
+    @drop_datasets
+    def test_update_samples(self):
```

Ruff (0.8.2)

Redefinition of unused `test_update_samples` from line 1068

(F811)